src/webui/www/webui.qrc

```diff
@@ -27,14 +27,13 @@
         <file>private/scripts/contextmenu.js</file>
         <file>private/scripts/download.js</file>
         <file>private/scripts/dynamicTable.js</file>
-        <file>private/scripts/lib/clipboard.min.js</file>
-        <file>private/scripts/lib/excanvas-compressed.js</file>
-        <file>private/scripts/lib/mocha-yc.js</file>
+        <file>private/scripts/lib/clipboard-2.0.0.min.js</file>
+        <file>private/scripts/lib/mocha-0.9.6-yc.js</file>
         <file>private/scripts/lib/mootools-1.2-core-yc.js</file>
         <file>private/scripts/lib/mootools-1.2-more.js</file>
-        <file>private/scripts/lib/parametrics.js</file>
         <file>private/scripts/misc.js</file>
         <file>private/scripts/mocha-init.js</file>
+        <file>private/scripts/speedslider.js</file>
```

Comment:
should be placed below `private/scripts/prop-webseeds.js`

Reply:
Forgot to update it after my replace all